core-bundle/src/EventListener/ForceTurboPrivateCacheListener.php

```diff
@@ -29,6 +29,13 @@ public function __invoke(ResponseEvent $event): void
             return;
         }
 
+        $request = $event->getRequest();
+
+        if ($request->headers->get('x-turbo-request-id')) {
+            $event->getResponse()->headers->set('Cache-Control', 'private, max-age=5, must-revalidate');
```

Comment:
We could add a simple `private readonly int $turboMaxAge = 5` to the constructor, in case anyone wants to adjust the service. But feel free not to :)

Reply:
I like that more

Reply:
See d8e4549dc6c2f96d61726966d151e13578b57042